__tests__/api.test.js

```diff
@@ -4,10 +4,23 @@ const db = require('../db/connection')
 const data = require('../db/data/test-data/index')
 const seed = require('../db/seeds/seed')
 const topics = require('../db/data/test-data/topics')
+const endpoints = require('../endpoints.json')
 
-beforeAll(() => seed(data))
+beforeEach(() => seed(data))
 afterAll(() => db.end())
 
+describe('/api', () => {
+    describe('GET', () => {
+        it('responds with an object containing a list of the available endpoints', () => {
+            return request(app)
+            .get('/api')
+            .expect(200)
+            .then(({body}) => {
+                expect(body).toEqual({endpoints})
+            })
+        })
+    })
+})
```

Comment:
I think another test checking the keys are the ones you expect :)

Reply:
I'm not sure how to do this and to get it to fail, and if I manually type it in each time I update the file I would need to update the test?